src/main/resources/Template.java

```diff
@@ -1,8 +1,7 @@
 package <package_name>;
 public class <project_name> {
 
+    public static void main(String[] args) {
 
-public static void main(String[]args) {
-
-        }
+    }
 }
```

Comment:
Not strictly relevant to this PR, but fixes the issue where the generator generates Java with weird indentation.